### ISSUE TITLE
Disable breakpoint command

### ISSUE THREAD

**Issue by clperez** [Microsoft/vscode#46509]
I don't think its possible to disable just one breakpoint using key bindings.
I would like to see it implemented.

**Comment by isidorn** [comment]
This is correct, for that a "Disable Breakpoint Command" would have to be registered in the quick open menu.
Code pointer where to add that command https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/debug/browser/debugCommands.ts#L29

We accept PRs that fix this
